packages/pages-shared/__tests__/asset-server/handler.test.ts

```diff
@@ -43,6 +50,20 @@ describe("asset-server handler", () => {
 
 		expect(response.status).toBe(302);
 		expect(response.headers.get("Location")).toBe("/home");
+
+		const { response: proxyResponse } = await getTestResponse({
+			request: "https://foo.com/200",
+			metadata,
+			findAssetEntryForPath: async (path: string) => {
```

Comment:
It _looks_ like this function doesn't return a legit `AssetEntry`? I recognize that the other versions of this function in this file are the same.

Does that make testing the content of `await proxyResponse.text()` too difficult? If yes, no biggie.

Reply:
I'm not sure of the exact way to write this (just copied the others as you said) - but I note that when we don't supply this function, the return status is 404, so checking the response returns 200 seems like a good check to me.

Reply:
Yeah, fair enough. No stress. We can keep it as-is. You've got it well covered in the fixture test, and that's good enough for me.